src/App.tsx

```diff
@@ -1,14 +1,32 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
 import React from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  query: string,
+};
 
-export class App extends React.Component<{}, State> {
-  state: State = {};
+export class App extends React.PureComponent<{}, State> {
+  state: State = {
+    query: '',
+  };
+
+  handleOnChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = event.target;
+    const key = name as keyof State;
+
+    this.setState({ [key]: value });
+  };
 
   render() {
+    const { query } = this.state;
+    const visibleMovies: Movie[] = [...moviesFromServer].filter(movie => (
+      movie.title.toLocaleLowerCase().includes(query.toLocaleLowerCase())
+      || movie.title.toLocaleLowerCase().includes(query.toLocaleLowerCase())
```

Comment:
![image](https://imgur.com/a/AGju6ny)
![image](https://user-images.githubusercontent.com/23210681/150919407-ead68a79-e9f0-4aa7-86c7-c169ad989a61.png)

Something went wrong in searching in upper case, I tried to find film with Washington in description